### PR TITLE
Sphinx documentation. Fixes #6

### LINKED ISSUE #6: Docs to RTD

### PR DESCRIPTION
**Pull request by jamadden**
Docs can be viewed at http://zopebrowserresource.readthedocs.io/en/issue6/

### CODE REVIEW

File: docs/overview.rst [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+===============
+
+There are two ways to traverse to a resource:
+
+1. with the 'empty' view on a site, e. g. ``http://localhost/@@/myfile``
+   (This is declared by ``zope.browserresource``'s ``configure.zcml``
+   to be `.Resources`.)
+
+2. with the ``++resource++`` namespace, e. g. ``http://localhost/++resource++myfile``
+   (This is declared by `zope.traversing.namespace`)
```
Comment (mgedmin) [inline docs/overview.rst]: Wait, wait, this actually works?!  TIL.

File: docs/overview.rst [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+Rather than putting together the URL to a resource manually, you should use
+`zope.traversing.browser.interfaces.IAbsoluteURL` to get the URL, or for a
+shorthand, call the resource object. This has an additional benefit:
+
+If you want to serve resources from a different URL, for example
+because you want to use a web server specialized in serving static
+files instead of the appserver, you can register an ``IAbsoluteURL``
+adapter for the site under the name ``resource`` that will be used to
+compute the base URLs for resources. (See `.AbsoluteURL` for the
+implementation.)
```
Comment (mgedmin) [inline docs/overview.rst]: Also TIL.

File: src/zope/browserresource/resources.py [hunk 3]
```diff
@@ -106,9 +119,10 @@ def browserDefault(self, request):
         return empty, ()
 
     def __getitem__(self, name):
-        '''A helper method to make this view usable from templates,
-        so resources can be acessed in template like context/@@/<resourcename>.
-        '''
+        """
+        A helper method to make this view usable from templates,
+        so resources can be accessed in template like ``context/@@/<resourcename>``.
```
Comment (mgedmin) [inline src/zope/browserresource/resources.py]: What?  TIL.

**Comment by jamadden** [comment]
Thank you!

**Pull request merged**